public/src/api/httpClient.ts

```diff
@@ -20,6 +20,18 @@ export function getTestimonialsCount(): Promise<any> {
   });
 }
 
+export function getProducts(): Promise<any> {
```

Comment:
Declare a proper return type, avoid using `any` type at all

Reply:
@derevnjuk This whole file uses the `any` return type for the same kind of functions. Should I update all the functions there?

Reply:
You can do it, but that's not necessary. The main purpose is to align new code according to our code style.